File: PioneerApp/src/Entities/Enfant.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.awt.Image;
import java.io.File;
import javax.swing.JFileChooser;
import javax.swing.filechooser.FileNameExtensionFilter;

/**
 *
 * @author Alfa Shel
 */
public class Enfant {
    
    private int id_e;
    private String nom;
    private String prenom;
    private String sexe;
    private String cin_p;
    private String age;
    private String photo;
    private Image icon;

    public Enfant(String nom, String prenom, String sexe, String cin_p, String age,String photo) {
        this.nom = nom;
        this.prenom = prenom;
        this.sexe = sexe;
        this.cin_p = cin_p;
        this.age = age;
        this.photo=photo;
//        JFileChooser file = new JFileChooser();
//        file.setCurrentDirectory(new File(System.getProperty("user.home")));
//        FileNameExtensionFilter filter = new FileNameExtensionFilter("*.Images","jpg","png","jpeg");
//        file.addChoosableFileFilter(filter);
//        int result = file.showSaveDialog(null);
//        if(result== JFileChooser.APPROVE_OPTION)
//        {
//            File selecteFile = file.getSelectedFile();
//             this.photo = selecteFile.getAbsolutePath();
//        }
    }
        
       public Enfant(int id_e,String nom, String prenom, String sexe, String cin_p, String age,Image icon) {
       
        this.id_e=id_e;
        this.nom = nom;
        this.prenom = prenom;
        this.sexe = sexe;
        this.cin_p = cin_p;
        this.age = age;
        this.icon=icon;
        
        
         }

    public int getId_e() {
        return id_e;
    }

    public void setId_e(int id_e) {
        this.id_e = id_e;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getSexe() {
        return sexe;
    }

    public void setSexe(String sexe) {
        this.sexe = sexe;
    }

    public String getCin_p() {
        return cin_p;
    }

    public void setCin_p(String cin_p) {
        this.cin_p = cin_p;
    }

    public String getAge() {
        return age;
    }

    public void setAge(String age) {
        this.age = age;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public Image getIcon() {
        return icon;
    }

    public void setIcon(Image icon) {
        this.icon = icon;
    }

    @Override
    public String toString() {
        return "Enfant{" + "id_e=" + id_e + ", nom=" + nom + ", prenom=" + prenom + ", sexe=" + sexe + ", cin_p=" + cin_p + ", age=" + age + '}';
    }
        
    
    
}

File: PioneersWeb/src/PioneerBundle/Entity/Enfant.php
<?php

namespace PioneerBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Enfant
 *
 * @ORM\Table(name="enfant", indexes={@ORM\Index(name="cin_p", columns={"cin_p"})})
 * @ORM\Entity
 */
class Enfant
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_e", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idE;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=32, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="prenom", type="string", length=32, nullable=false)
     */
    private $prenom;

    /**
     * @var string
     *
     * @ORM\Column(name="sexe", type="string", length=10, nullable=false)
     */
    private $sexe;

    /**
     * @var string
     *
     * @ORM\Column(name="cin_p", type="string", length=8, nullable=false)
     */
    private $cinP;

    /**
     * @var string
     *
     * @ORM\Column(name="age", type="string", length=1, nullable=false)
     */
    private $age;

    /**
     * @var string
     *
     * @ORM\Column(name="photo", type="string", length=255, nullable=false)
     */
    private $photo;

    /**
     * @return int
     */
    public function getIdE()
    {
        return $this->idE;
    }

    /**
     * @param int $idE
     */
    public function setIdE($idE)
    {
        $this->idE = $idE;
    }

    /**
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * @param string $nom
     */
    public function setNom($nom)
    {
        $this->nom = $nom;
    }

    /**
     * @return string
     */
    public function getPrenom()
    {
        return $this->prenom;
    }

    /**
     * @param string $prenom
     */
    public function setPrenom($prenom)
    {
        $this->prenom = $prenom;
    }

    /**
     * @return string
     */
    public function getSexe()
    {
        return $this->sexe;
    }

    /**
     * @param string $sexe
     */
    public function setSexe($sexe)
    {
        $this->sexe = $sexe;
    }

    /**
     * @return string
     */
    public function getCinP()
    {
        return $this->cinP;
    }

    /**
     * @param string $cinP
     */
    public function setCinP($cinP)
    {
        $this->cinP = $cinP;
    }

    /**
     * @return string
     */
    public function getAge()
    {
        return $this->age;
    }

    /**
     * @param string $age
     */
    public function setAge($age)
    {
        $this->age = $age;
    }

    /**
     * @return string
     */
    public function getPhoto()
    {
        return $this->photo;
    }

    /**
     * @param string $photo
     */
    public function setPhoto($photo)
    {
        $this->photo = $photo;
    }


}

